chat_shell/chat_shell/tools/knowledge_injection_strategy.py

```
@@ -220,13 +231,22 @@ def format_chunks_for_injection(
         for i, chunk in enumerate(chunks):
             content = chunk.get("content", "")
             source = chunk.get("source", "Unknown")
-            score = chunk.get("score", 0.0)
+            score = chunk.get("score")
             kb_id = chunk.get("knowledge_base_id", 0)
```

Ruff (0.14.11)

Local variable `kb_id` is assigned to but never used

Remove assignment to unused variable `kb_id`

(F841)